java/org/contikios/cooja/plugins/analyzers/IPHCPacketAnalyzer.java

```diff
@@ -271,33 +259,29 @@ public int analyzePacket(Packet packet, StringBuilder brief,
         } else {
           /* non-context based multicast compression */
           switch (packet.get(1) & SICSLOWPAN_IPHC_DAM_11) {
-            case SICSLOWPAN_IPHC_DAM_00: /* 128 bits */
+            case SICSLOWPAN_IPHC_DAM_00 -> {
               /* copy whole address from packet */
-
               packet.copy(hc06_ptr, destAddress, 0, 16);
               hc06_ptr += 16;
-              break;
-            case SICSLOWPAN_IPHC_DAM_01: /* 48 bits FFXX::00XX:XXXX:XXXX */
-
+            }
+            case SICSLOWPAN_IPHC_DAM_01 -> {
```

Comment:
Are you deliberately removing the comments after the colon? You removed some sizes in the switch dealing with `SICSLOWPAN_IPHC_SAM_10` a page or two earlier.

If the comments are bad I think it's better to remove them, just double checking that it wasn't IntelliJ that wiped them for you.